conans/client/loader.py

```diff
@@ -362,6 +366,7 @@ def _load_python_file(conan_file_path):
                 else:
                     if folder.startswith(current_dir):
                         module = sys.modules.pop(added)
+                        module.print = new_print
```

Comment:
I like this - but makes me wonder if there are any risks since `print` is a built-in? thinking of unusual scenarios like a frozen interpreter embedded in an executable

Reply:
I had a look, and in theory it seems ok, this kind of overwrite of module globals seems allowed, it is not changing the built-in print, just making the module ``print()`` pointing to another function, and this is done per-module, not globally.